src/app/test-record/test-record.mapper.ts

```diff
@@ -0,0 +1,288 @@
+        66: 'Part paid first test retest',
+        67: 'Free first test retest',
+        98: 'Paid annual test retest',
+        99: 'Part paid annual test retest',
+        103: 'Paid first test retest',
+        104: 'Part paid first test retest',
+        70: 'Paid prohibition clearance (full inspection with certification)',
+        76: 'Paid prohibition clearance (retest with certification)',
+        79: 'Part paid prohibition clearance (retest with certification)',
+        82: 'Paid prohibition clearance on first test (full inspection with certification)',
+        83: 'Paid retest prohibition clearance on first test',
+        107: 'Paid prohibition clearance (full inspection with certification)',
+        113: 'Paid prohibition clearance (retest with certification)',
+        116: 'Part paid prohibition clearance (retest with certification)',
+        119: 'Paid prohibition clearance on first test (full inspection with certification)',
+        120: 'Paid retest prohibition clearance on first test',
+        122: 'Voluntary roadworthiness test HGV/TRL',
+        91: 'Voluntary roadworthiness test TRL'
+      },
+      testSectionApplicable2: {
+        30: 'Voluntary brake test',
+        31: 'Voluntary headlamp aim check',
+        32: 'Voluntary smoke test',
+        33: 'Voluntary multi check (headlamp aim, smoke and brake test)',
+        34: 'Voluntary speed limiter check',
+        35: 'Voluntary Vitesse 100',
+        36: 'Voluntary Tempo 100',
+        38: 'Notifiable alteration check',
+        39: 'Low Emissions Certificate (LEC) with annual test',
+        44: 'Low Emissions Certificate (LEC) with annual test',
+        45: 'Low Emissions Certificate (LEC)',
+        47: 'Free notifiable alteration',
+        48: 'Paid notifiable alteration',
+        49: 'TIR test',
+        50: 'ADR test',
+        56: 'Paid TIR retest',
+        57: 'Free TIR retest',
+        59: 'Paid ADR retest',
+        60: 'Free ADR retest',
+        85: 'Voluntary brake test',
+        86: 'Voluntary multi-check',
+        87: 'Voluntary Shaker plate check',
+        88: 'Voluntary Speed limiter check',
+        89: 'Voluntary smoke test',
+        90: 'Voluntary headlamp aim test',
+        100: 'Vitesse 100 Replacement',
+        121: 'Vitesse 100 Application'
+      }
+    };
+
+    return testTypesApplicable[sectionName];
+  }
+
+  getCountryOfRegistrationCode(countryName: string) {
+    const countryCode = Object.keys(COUNTRY_OF_REGISTRATION)[
+      Object.values(COUNTRY_OF_REGISTRATION).indexOf(countryName)
+    ];
+    return !!countryCode ? countryCode : '';
+  }
+
+  mapFormValues(testResultFormData, testResultObject: TestRecordTestType): TestResultModel {
+    const testResultMapped: TestResultModel = testResultObject.testRecord;
+    const testTypeMapped: TestType = testResultObject.testType;
+    const testTypesApplicable = {
+      seatBeltApplicable: this.getTestTypeApplicable('seatBeltApplicable'),
```

Comment:
Can we expand the unit tests for those?
I think at the moment we only check for one but we seem to have many more than one